src/scss/style.scss

```diff
@@ -148,17 +148,23 @@ nav {
     top: 0;
     left: 0;
     height: 100%;
-    width: 250px;
+    width: 15.625rem;
     background-color: $light-mode-nav-bar-color;
-    border-right: 1px solid $light-mode-border-color;
-    @include cross-browser-transition-set-long ($transition-05 !important);
+    border-right: .0625rem solid $light-mode-border-color;
+    transition:                 $transition-05 !important;
+    -webkit-transition:         $transition-05 !important;
+    -moz-transition:            $transition-05 !important;
+    -o-transition:              $transition-05 !important;
     &.close {
-        width: 73px !important;
+        width: 4.5625rem !important;
         ~ .dashboard .top,
         ~ .dashboard .dashboard-content {
-            left: 73px;
-            width: calc(100% - 73px);
-            @include cross-browser-transition-set-long ($transition-05 !important);
+            left: 4.5625rem;
+            width: calc(100% - 4.5625rem);
+            transition:                 $transition-05 !important;
```

Comment:
Lines 164 - 167 have replaced the `@include cross-browser-transition-set-long ($transition-05 !important)` directive owing to the build error invoked due to a +1 argument passed (default of 0.)

Reply:
Accepted substitute